protected/humhub/modules/ui/menu/MenuEntry.php

```diff
@@ -230,6 +231,18 @@ public function setIsVisible($isVisible)
         return $this;
     }
 
+    /**
+     * Returns the value of the isVisible property. If the value is not set, null is returned.
+     * To query visibility, use the method `isVisible()`.
+     *
+     * @return bool|null
+     * @since 1.8
+     */
+    public function getIsVisible()
```

Comment:
I'd prefere something like `isVisibilitySet()` or `hasVisibilityState()` and document that this may be used in menues to be able to set a default value. I think there is no other use case for directly determining the value of `isVisible` since we only have three states `uninitialized`, `true`, `false`. But its your call in case you prefere the current solution.

Reply:
Agreed, see commit: 06bacae